src/components/ecl-indicator/ecl-builder.config.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+const pkg = require('./package.json');
+
+const nodeModules = path.resolve(__dirname, './node_modules');
+const outputFolder = './build';
+
+const includePaths = [nodeModules];
+
+const banner = `${pkg.name} - ${pkg.version} Built on ${new Date().toISOString()}`;
+
+module.exports = {
+  scripts: [
+    {
+      entry: path.resolve(nodeModules, '@ecl/indicator/indicator.js'),
+      dest: path.resolve(outputFolder, 'scripts/ecl-indicator.js'),
+      options: {
+        banner,
```

Comment:
we need to set the format to "iife" here, otherwise we cannot call the init, at least for now..